webapp/sources/rudder/rudder-web/src/main/elm/rules/sources/ViewTabContent.elm

```diff
@@ -444,33 +450,52 @@ tabContent model details =
             let
               groupTreeElem : Group -> Html Msg
               groupTreeElem item =
-                    li [class "jstree-node jstree-leaf"]
-                    [ i[class "jstree-icon jstree-ocl"][]
-                    , a[href "#", class "jstree-anchor"]
-                      [ i [class "jstree-icon jstree-themeicon fa fa-sitemap jstree-themeicon-custom"][]
-                      , span [class "treeGroupName tooltipable"][text item.name, (if item.dynamic then (small [class "greyscala"][text "- Dynamic"]) else (text ""))]
-                      , div [class "treeActions-container"]
-                        [ span [class "treeActions"][ span [class "tooltipable fa action-icon accept", onClick (SelectGroup (NodeGroupId item.id) True)][]]
-                        , span [class "treeActions"][ span [class "tooltipable fa action-icon except", onClick (SelectGroup (NodeGroupId item.id) False)][]]
-                        ]
+                let
+                  checkIncludeOrExclude : List RuleTarget -> Bool
+                  checkIncludeOrExclude lst = lst
+                    |> List.map (\t -> case t of
+                       NodeGroupId groupId -> groupId
+                       Composition _ _ -> "compo"
```

Comment:
I Think this need to be a recursive function and check in And/Or/Composition Element to see if the item is inside